explore-analyze/dashboards/add-controls.md

```diff
@@ -116,6 +116,60 @@ Change the settings for the Options list and Range slider controls.
 1. Hover over the control you want to edit, then click ![The Edit control icon that opens the Edit control flyout](../../images/kibana-dashboard_controlsEditControl_8.3.0.png "").
 2. On the **Edit control** flyout, change the options, then click **Save and close**.
 
+## Add controls from your ES|QL visualization's query [add-esql-control]
+
+You can bind controls to your ES|QL visualizations in dashboards. When creating an ES|QL visualization, the autocomplete suggestions 
+prompt control insertion for field values, field names, function configuration, and function names. This enables controls that only 
```

Comment:
Function names added in 8.19 so let's not mention here!

Reply:
```suggestion
prompt control insertion for field values values and field names. This enables controls that only 
```
Like this?